pkg/metrics/statsd/statsd_test.go

```diff
@@ -83,3 +86,247 @@ func TestEmitFloat(t *testing.T) {
 		t.Error(m)
 	}
 }
+
+func TestEmitGaugeViaUDP(t *testing.T) {
```

Comment:
I think we can drop this test: we are basically testing Go's standard library here. The only difference between unix socet and UDP from our point of view is what we pass into `net.Dial`. Everything else is `net` package implementation which is tested in it's own.

What we might want to do instead is this: make `net.Dial` mockable and check what we pass into into. In `statsd` struct we are already making `time.Now()` mockable here:

https://github.com/Azure/ARO-RP/blob/1ae6115606b60c4f530ae4154cae8f734e161cc2/pkg/metrics/statsd/statsd.go#L44

It will be the same appraoch.

But the value of this test is not very high given that we are testing `getConnectionDetails()` separately and there is no extra logic in `dial()`. So IMO we can just drop this test.

Reply:
Dropped the test.